docs/installation_configuration_guide/ModelConfiguration.md

```diff
@@ -189,14 +189,15 @@ Errors can be sent to specified Kafka topic in following json format (see below
 to see where they should be set):
 
-| Name                  | Default value | Description                                                                                                                                                                                |
-| ------------          | ------------- | ------------                                                                                                                                                                               |
-| topic                 | -             | Topic where errors will be sent. It should be configured separately (or topic `auto.create` setting should be enabled on Kafka cluster)                                                    |
-| stackTraceLengthLimit | 50            | Limit of stacktrace length that will be sent (0 to omit stacktrace at all)                                                                                                                 |
-| includeHost           | true          | Should name of host where error occurred (e.g. TaskManager in case of Flink) be included. Can be misleading if there are many network interfaces or hostname is improperly configured)     |
-| includeInputEvent     | false         | Should input event be serialized (can be large or contain sensitive data so use with care)                                                                                                 |
-| useSharedProducer     | false         | For better performance shared Kafka producer can be used (by default it's created and closed for each error), shared Producer is kind of experimental feature and should be used with care |
-| additionalParams      | {}            | Map of fixed parameters that can be added to Kafka message                                                                                                                                 |
+| Name                   | Default value | Description                                                                                                                                                                                |
+|------------------------|---------------| ------------                                                                                                                                                                               |
+| topic                  | -             | Topic where errors will be sent.                                                     |
+| createTopicIfNotExists | true          | Should error topic be created if it does not exist. Please note that it will be created using default configs - for production deployments make sure the config is ok. Set to false if e.g. Kafka user for Nussknacker does not have permissions to create (in that case the topic has to be configured manually) |
```

Comment:
maybe let's add notice, that in such case Nussknacker will start normally and accept scenario deployments, but those are gonna fail.